Tests/AfluentTests/WaitUntilCondition.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  WaitUntilCondition.swift
+//  Afluent
+//
+//  Created by Annalise Mariottini on 11/9/24.
+//
+
+import Afluent
+
+/// Waits for some condition to proceeds, unless the specified timeout is reached, in which case an error is thrown.
```

Comment:
Minor grammar issue here with "proceeds". It's also maybe a little strange phrasing with "proceed".

Reply:
Sorry for so many docc typos in this PR 🤦